oshi-core/src/main/java/oshi/util/ExecutingCommand.java

```diff
@@ -81,7 +83,16 @@ private ExecutingCommand() {
     public static List<String> runNative(String[] cmdToRunWithArgs) {
         Process p = null;
         try {
-            p = Runtime.getRuntime().exec(cmdToRunWithArgs);
+            PlatformEnum platform = SystemInfo.getCurrentPlatformEnum();
+            if (platform == PlatformEnum.WINDOWS) {
+                p = Runtime.getRuntime().exec(cmdToRunWithArgs, new String[]{"LANGUAGE=C"});
+            } else if (platform == PlatformEnum.LINUX || platform == PlatformEnum.FREEBSD
+                    || platform == PlatformEnum.SOLARIS || platform == PlatformEnum.AIX
+                    || platform == PlatformEnum.MACOSX) {
+                p = Runtime.getRuntime().exec(cmdToRunWithArgs, new String[]{"LC_ALL=C"});
```

Comment:
Let's make this string array a constant too.  Actually you can probably just define a single constant `EXEC_ENV` (or similar) and use the platform conditional to decide which array it is. The platform won't change during execution.  Then you can just use that as the second arg here with no conditional.